OrbitLinuxTracing/LinuxTracingUtils.cpp

```diff
@@ -122,6 +123,12 @@ std::optional<char> GetThreadState(pid_t tid) {
 
   std::string first_line{};
   std::getline(stream, first_line);
+
+  // Remove the process name enclosed in parentheses in the second position
+  // as it could contain spaces.
+  first_line = std::regex_replace(first_line, std::regex{"\\(.*\\)"}, "()",
+                                  std::regex_constants::format_first_only);
```

Comment:
Since `std::regex` is known to be slow I was wondering if something `std::string_view{first_line}.substr(first_line.find(')')` wouldn't be enough to get rid of the problem. Of course error handling needs to be added and the indexes need adjustment. WDYT?

Nice catch, by the way!

Reply:
Makes sense, I guess a regex is a bit overkill here. I'll switch away from it.

Reply:
I've done what you suggested, please have a look as the code is obviously a bit uglier now.

Reply:
I like it. Your verbose variable names make it pretty obvious what's happening. At least I believe so - obviously I'm now already familiar with what needed to be solved.